<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>隶书</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>777</width>
         <height>329</height>
        </rect>
       </property>
       <attribute name="label">
        <string>新品入库</string>
       </attribute>
       <widget class="QToolBox" name="toolBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-30</y>
          <width>781</width>
          <height>421</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>777</width>
           <height>359</height>
          </rect>
         </property>
         <attribute name="label">
          <string>新品入库</string>
         </attribute>
         <widget class="QComboBox" name="newCategoryComboBox">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>81</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>62</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>类别：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>41</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>名称：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="newNameLineEdit">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>60</y>
            <width>191</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>41</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>进价：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>100</y>
            <width>51</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>售价：</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="newInputPriceLineEdit">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>100</y>
            <width>61</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="newOutputPriceLineEdit">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>100</y>
            <width>61</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>140</y>
            <width>41</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>数量：</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="newCountSpinBox">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>140</y>
            <width>61</width>
            <height>27</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
         <widget class="QPushButton" name="newUploadPushButton">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>140</y>
            <width>88</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>上传</string>
          </property>
         </widget>
         <widget class="QLabel" name="newPictureLabel">
          <property name="geometry">
           <rect>
            <x>131</x>
            <y>180</y>
            <width>141</width>
            <height>131</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>190</y>
            <width>62</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>商品样照：</string>
          </property>
         </widget>
         <widget class="QPushButton" name="newPutinStorePushButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>88</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>入库</string>
          </property>
         </widget>
         <widget class="QPushButton" name="newClearancePushButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>280</y>
            <width>88</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>清仓</string>
          </property>
         </widget>
         <widget class="QTableView" name="commodityTableView">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>10</y>
            <width>441</width>
            <height>311</height>
           </rect>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </widget>
        <widget class="QWidget" name="widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>28</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 2</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>777</width>
         <height>329</height>
        </rect>
       </property>
       <attribute name="label">
        <string>预售订单</string>
       </attribute>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>类别：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>名称：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>单价：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>库存：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>总价：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="preCategoryComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>101</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="preNameComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>171</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="preOutputPriceLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="preAmountLabel">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>90</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="preTotalLabel">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>130</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="preCountSpinBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>126</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="prePictureLabellabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="preShellPushButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>250</y>
          <width>88</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>出售</string>
        </property>
       </widget>
       <widget class="QPushButton" name="prePlaceOrderPushButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>290</y>
          <width>88</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>下单</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>0</y>
          <width>441</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
